apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -3182,6 +3182,8 @@ static void mapEobCommonGroupInpOutHHAHospiceSNF(
     if (fiscalIntermediaryNumber.isPresent()) {
       eob.addExtension(
           createExtensionCoding(eob, CcwCodebookVariable.FI_NUM, fiscalIntermediaryNumber));
+      eob.addExtension(
+          createExtensionIdentifier(CcwCodebookVariable.FI_NUM, fiscalIntermediaryNumber));
```

Comment:
@JFU-NAVA-PBC V1 sets the FI_NUM only as an extension identifier but I see that for V2 here you're setting it in 2 spots. Is that fine? Wouldn't that be duplicative?